### ISSUE TITLE
filenames recorded in poses_aligned.csv do not exist

### ISSUE THREAD

**Issue by guzhx** [csiro-robotics/Wild-Places#9]
Thank you for your admirable work! I intend to train and evaluate my model on your dataset. However, after generating training sets, it seems that about 1118 filenames recorded in poses_aligned.csv in K_01 and K_02 of Karawatha are inconsistent with existing file. For example, as shown in figures below.
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=6954316540469768267 skey=@crypt_624bd187_fad5e01921acd0ae7bdd7546f95d13a1 mmweb_appid=wx_webfilehelper](https://github.com/csiro-robotics/Wild-Places/assets/112327023/fe00c25f-4ba3-44d9-881a-f3d6d6d14c7e)
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=4015034238257687477 skey=@crypt_624bd187_fad5e01921acd0ae7bdd7546f95d13a1 mmweb_appid=wx_webfilehelper](https://github.com/csiro-robotics/Wild-Places/assets/112327023/06247820-2c54-4f3c-85b4-29e6725fa177)
I wonder if there's something wrong here?

**Comment by shenyanqing1105** [comment]
I think we should modify the filenmaes in pose_aligned.csv @guzhx

**Comment by shenyanqing1105** [comment]
> I think we should modify the filenmaes in pose_aligned.csv @guzhx

and then generate pickle files for evaluation

**Comment by a-github-learner** [comment]
Thank you for your work! Have you ever fixed the inconsistency of pose_aligned.csv? @syqlyx

**Comment by Rocense** [comment]
Thank you for your great work! Have you fixed the inconsistency between pcd filenames and that in pose_aligned.csv?@syqlyx

**Comment by peymmo** [comment]
@kavisha725 Could you please assist with this inquiry?

**Comment by kavisha725** [comment]
@JBKnights you we in charge of the file upload, right? Can you please have a look?

**Comment by shenyanqing1105** [comment]
> Thank you for your great work! Have you fixed the inconsistency between pcd filenames and that in pose_aligned.csv?@syqlyx

yes, I just read the filenames from the folder and then replaced them in csv files. I mean, only the last few digits of the file names don’t match.

**Comment by JBKnights** [comment]
Fixed this issue by uploading new pose_aligned.csv with fixed timestamps, closing
